test-cases/PR-provision-test.yaml

```diff
@@ -19,7 +19,7 @@ instance_type_db: 'i4i.large'
 n_monitor_nodes: 1
 n_db_nodes: 3
 
-nemesis_class_name: NonDisruptiveMonkey
+nemesis_class_name: MgmtRepair
```

Comment:
keep in mind to remove revert this

Reply:
We should document open issues for those things

We can switch to not doing any nemesis, if we want to focus on provision

We could add labels for running some of the individual nemesis